frontend/app/src/people/widgetViews/AboutView.tsx

```diff
@@ -258,7 +258,7 @@ export const AboutView = observer((props: AboutViewProps) => {
 
       <Divider />
       <Row>
-        <div>Price to Connect:</div>
+        <div>Price to Join:</div>
```

Comment:
@ecurrencyhodler this wording is incorrect we should keep it as connect cause it's weird to say 'join' as adding another person as a contact

Reply:
@kevkevinpal updated here [7f86e34](https://github.com/stakwork/sphinx-tribes/pull/923/commits/7f86e34f1841e3b5cf1af858b51e134e9aca0c49)

Reply:
Good catch thanks kev.